catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -339,10 +339,11 @@ public void addFields(Metacard metacard, SolrInputDocument solrInputDocument)
       try {
         byte[] luxXml = createTinyBinary(metacard.getMetadata());
         solrInputDocument.addField(LUX_XML_FIELD_NAME, luxXml);
-      } catch (XMLStreamException | SaxonApiException | IOException e) {
+      } catch (Exception e) {
         LOGGER.debug(
             "Unable to parse metadata field.  XPath support unavailable for metacard {}",
```

Comment:
Do you think we should add this warning onto the metacard itself so it's easier to track which metacards are losing the xpath support?

Reply:
I considered that, but it's a little more difficult because we're already storing the metacard into a solr document here. I'm also not sure that we should be modifying metacards at this point, storage and retrieval is expected to be transparent.